### PR TITLE
Refactor init and push commands

### PR DESCRIPTION
**Pull request by NatashaWso2**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/init/models/SrcFile.java [hunk 1]
```diff
@@ -28,18 +28,27 @@
...
+                                                  "documentation {\n" +
+                                                  "   A service is a network-accessible API\n" +
+                                                  "   Advertised on '/hello', port comes from listener endpoint\n" +
+                                                  "}\n" +
                                                   "service<http:Service> hello bind listener {\n" +
                                                   "\n" +
-                                                  "    // A resource is an invokable API method\n" +
-                                                  "    // Accessible at '/hello/sayHello\n" +
-                                                  "    // 'caller' is the client invoking this resource \n" +
+                                                  "    documentation{\n" +
```
Comment (hemikak) [inline cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/init/models/SrcFile.java]: Space between documentation and {

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/init/models/SrcFile.java [hunk 2]
```diff
@@ -56,25 +65,34 @@
                                                   "}";
 
     private static final String MAIN_FUNCTION_CONTENT = "import ballerina/io;\n" +
+                                                        "documentation {\n" +
```
Comment (hemikak) [inline cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/init/models/SrcFile.java]: Should there be an empty line between the import and the documentation

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/init/models/SrcFile.java (hunk 1 above)
Comment (NatashaWso2) [reply cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/init/models/SrcFile.java]: Fixed with 0af28ab

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/init/models/SrcFile.java (hunk 2 above)
Comment (NatashaWso2) [reply cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/init/models/SrcFile.java]: Added an empty line with 0af28ab

**Pull request merged**